docs/specs/restore.yml

```diff
@@ -3,9 +3,25 @@
 inputs:
   docfx.yml: |
     dependencies:
-      docfx-test-dependencies: https://github.com/docascode/docfx-test-dependencies
+      docfx-test-dependencies1: https://github.com/docascode/docfx-test-dependencies
+      docfx-test-dependencies2: https://github.com/docascode/docfx-test-dependencies
 restores:
   git/github.com/docascode/docfx-test-dependencies/9ab39126f6d6642131084e741ca6206989db505a/dep.md: DEP
   git/github.com/docascode/docfx-test-dependencies/5736374c0a142252063bbedafe5d98db495e263d/dep.md: child-DEP
+  restore_lock/01. Restore dependencies with their children-*-lock.json: |
+    {"git":{"https://github.com/docascode/docfx-test-dependencies":"9ab39126f6d6642131084e741ca6206989db505a"}}
+  restore_lock/9ab39126f6d6642131084e741ca6206989db505a-*-lock.json: |
+    {"git":{"https://github.com/docascode/docfx-test-dependencies#child":"5736374c0a142252063bbedafe5d98db495e263d"}}
 outputs:
   build.manifest:
+---
+# Restored dependency repo can't be found
+restore: false
```

Comment:
what about use "commands: build", to indicate we don't need restore. "restore" and "restores" are a bit confusing.

Reply:
good suggestion

---
In reply to: [201568836](https://github.com/dotnet/docfx/pull/3042#discussion_r201568836) [](ancestors = 201568836)